README.rst

```diff
@@ -0,0 +1,41 @@
+|BuildStatus|_ |TotalDownloads|_ |LatestStableVersion|_ |License|_ |TYPO3|_
+
+.. |BuildStatus| image:: https://github.com/FriendsOfTYPO3/extension_builder/workflows/tests/badge.svg
```

Comment:
gibs da nicht auch ein badge für v10?

Reply:
Hab ich in den letzten commit `[DOCS] Adapt links and text to Extension Builder v10` integriert.